classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_reader :id
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+  end
+end
+
+class Classroom < Student::Base
+  has_many :student, class_name: 'Classroom'
+  belongs_to :classroom, class_name: 'Student'
+end
```

Comment:
- [ ] `*Check for a method that serves as the 'has-many' side (a classroom has many students).`

`def add_student(student)`

The add_student method should take a student as an argument.